pkg/sfu/downtrack.go

```diff
@@ -1567,6 +1568,34 @@ func (d *DownTrack) handleRTCP(bytes []byte) {
 					sal.OnTransportCCFeedback(d, p)
 				}
 			}
+
+		case *rtcp.ExtendedReport:
+			// SFU only reponse with the DLRRReport for the track has the sender SSRC, the behavior is different with
+			// browser's implementation that includes all the sent tracks. It is ok since all the tracks
+			// use the same connection, and server-sdk-go can get the rtt from the first DLRRReport
+			// (libwebrtc/browsers don't send XR to calculate rtt, it only responds)
```

Comment:
nit: spelling

```suggestion
			// SFU only responds with the DLRRReport for the track has the sender SSRC, the behavior is different with
			// browser's implementation, which includes all sent tracks. It is ok since all the tracks
			// use the same connection, and server-sdk-go can get the rtt from the first DLRRReport
			// (libwebrtc/browsers don't send XR to calculate rtt, it only responds)
```